README.md

```diff
@@ -309,11 +312,11 @@ After checking the integrity of the did document we need to make sure that the u
+Using only the already defined parameters from the did-core, we don't need to add extra fields to the did document. So being compliant to the schemas `"https://www.w3.org/ns/did/v1", "https://w3id.org/security/suites/jws-2020/v1"` that are used by the `did:web`, we are not forced to create a new did method like `did:webplus` It also reduces the the required requests we need to make to the server, since we only request one version of the did document and its metadata.
 
-### Using the DID as the subject id
+### Using the DID as the holder of verifiable credential
 
-The method can also be used as a subject inside a verifiable credential. Since we can make key rotation, the identifier has to include a reference to the version with `versionTime` and `versionId` and also the hashlink for integrity protection. As long as there is no newer version of the did document, the verifier just requests the did document and the metadata in the same way when the method is used as an issuer. But in case the holder rotated its keys, it is not able to proof the ownership with one of the keys that where in the linked version when setting the subject. It only has access to the current private key of the last did document version, so we need a connection from the linked version to the latest one. A new did document does not know the content of the next version, so we can not prelink it via a hash. But we are able to sign a credential with claiming what is the next did document version. The credential is a JSON Web Token, including the hash of the next version and signed by the current version:
+The method can also be used for the holder binding when issuing a credential. Since did:web allows to update the did document, the holder is able to perform key rotations to get rid of compromised keys. Therefore the identifier has to include a reference to the version with `versionTime` and `versionId` and also the hashlink for integrity protection. As long as there is no newer version of the did document, the verifier just requests the did document and the metadata in the same way when the method is used as an issuer. But in case the holder rotated his keys, he is not able to proof the ownership with one of the keys that where in the linked version when setting the subject. It only has access to the current private key of the last did document version, so we need a connection from the linked version to the latest one. A new did document does not know the content of the next version, so we can not prelink it via a hash. But we are able to sign a credential with claiming what is the next did document version. The credential is a JSON Web Token, including the hash of the next version and signed by the current version:
```

Comment:
I get what you want to say, but it seems to be formulated in a hard to read way.

Also, I would challenge the Part:

> Therefore the identifier has to include a reference to the version with `versionTime` and `versionId` and also the hashlink for integrity protection.


How about:

```suggestion
The method can also be used for the holder binding when issuing a credential. Since did:web allows to update the did document, the holder is able to perform key rotations to get rid of compromised keys. When binding a credential to a holder (or more generally using the holder's did as a reference to the holder anywhere) the un-versioned did should be used, exactly as did:web is used today. When the identity of a holder (or any person) at the present time of the validation is to be checked, sending the did without a version specification and hence receiving the most up to date did document is appropriate.

Fort added security, the issuer of a holder-bound VC might add a version + hash link parameter to the did and use this as the subject id, hence signing that he checked a certain version of the did document to belong to the holder did. This enables the verifier to check that their is a certificate chain going back from the current version of the did document to the one that was committed in the VC. This chain of certificats is build of JWTs, each one signed with a valid key from the old did document and refering to the next did doc via a hash link:
```